### ISSUE TITLE
about the matrix size and count

### ISSUE THREAD

**Issue by xiaoqi91** [YongyuG/rnnoise_16k#15]
I use denoise.c to extract features. And I changed count to 100000, so it printed 100000*75 on the screen. However when I run bin2hdf5.py , an error occurs: "cannot reshape array of size 7555881 into shape (100000,75)" . Why? looking forward to your reply, thank you.

**Comment by shipleyxie** [comment]
Have you soloved your problem yet? If so, could you paste your solution?

**Comment by xiaoqi91** [comment]
> Have you soloved your problem yet? If so, could you paste your solution?

yes, just because the strings from "printf" function are also saved to the binary file "training_16k_v3.f32". Just comment the "printf"s in denoise.c will work.

**Comment by YongyuG** [comment]
> > Have you soloved your problem yet? If so, could you paste your solution?
> 
> yes, just because the strings from "printf" function are also saved to the binary file "training_16k_v3.f32". Just comment the "printf"s in denoise.c will work.

sry for this silly mistake, will clear it soon
